src/kajal/BankingSystem.java

```diff
@@ -0,0 +1,76 @@
+package kajal;
+
+public class BankingSystem {
+	String name;
+	int debitcount;
+	int creditcount;
+	int printcount = 0;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value. and this balance should not hard coded.